codebuds/mattermost-publication-bundle/0.3/config/packages/mattermost_publication.yaml

```diff
@@ -0,0 +1,5 @@
+mattermost_publication:
+    webhook_url: '%env(MATTERMOST_WEBHOOK_URL)%'
+    channel: null
```

Comment:
You dont need all these null values. 

You could add a link to your bundles documentation instead.

Reply:
Hey, thanks for checking this out. I will do that.

Reply:
Ok I have just pushed the changes